tensorflow_asr/models/encoders/conformer.py

```diff
@@ -179,7 +183,6 @@ def __init__(self,
     def call(self, inputs, training=False, **kwargs):
         outputs = self.ln(inputs, training=training)
         B, T, E = shape_util.shape_list(outputs)
```

Comment:
we should remove this line after moving to conv1d

Reply:
and the other line below where `outputs = tf.reshape(outputs, [B, T, E])`

Reply:
Done